core/src/main/java/bio/terra/pearl/core/service/workflow/EnrollmentService.java

```diff
@@ -157,6 +151,34 @@ public HubResponse<Enrollee> enroll(PortalParticipantUser operator,
         return hubResponse;
     }
 
+    private Enrollee findOrCreateEnrolleeForEnrollment(ParticipantUser user, PortalParticipantUser ppUser, StudyEnvironment studyEnv, String studyShortcode, UUID preEnrollResponseId, boolean isSubjectEnrollment) {
+        return enrolleeService
+                .findByParticipantUserIdAndStudyEnv(user.getId(), studyShortcode, studyEnv.getEnvironmentName())
+                .filter(e -> {
```

Comment:
let's find a better name for this variable than "e" since that usual is used for "error".  any reason this can't be 'enrollee'?